haskell/private/packages.bzl

```diff
@@ -2,18 +2,20 @@
 
 load(":private/set.bzl", "set")
 
-def pkg_info_to_ghc_args(pkg_info):
+def pkg_info_to_ghc_args(pkg_info, for_plugin = False):
     """
     Takes the package info collected by `ghc_info()` and returns the actual
     list of command line arguments that should be passed to GHC.
+    Last argument indicates whether the package is a plugin dependency.
```

Comment:
Please be specific in the name of the argument, instead of "Last argument". This will prevent lost of sync between comment and code when a new argument will be added to the function.